ext/storage_sources/s3_store/README.md

```diff
@@ -0,0 +1,84 @@
+# WiredTiger's S3 Extension
+
+## Building and running 
+
+This is a guide to build WiredTiger with the S3 extension enabled.
+
+There are two different ways to build WiredTiger with S3 extension:
+*    1. Using a system installation of the S3.
```

Comment:
You are using both a bullet point and a number here - maybe remove the bullet points?

Reply:
The bullet point formats the number to be bullet point / list style, but happy to remove it and leave the numbers as regular text style.